### ISSUE TITLE
Optimization for loading

### ISSUE THREAD

**Issue by yongkangchen** [AtomLinter/Linter#130]
![2014-07-11 12 53 40](https://cloud.githubusercontent.com/assets/704762/3542049/f056f258-0852-11e4-85a9-78cfdc52cc1d.png)

It would be greatful that can use activationEvents to delay loading.
```
activationEvents (Optional): an Array of Strings identifying events that trigger your package's activation. You can delay the loading of your package until one of these events is triggered.
```
